src/jsonSchema.ts

```diff
@@ -88,6 +89,7 @@ export const jsonSchemaResolver = <TFieldValues extends Record<string, any>>(
       }).compile(validationSchema);
     }
 
+    // @ts-ignore
```

Comment:
What was the error here? Btw for future reference using `@ts-expect-error` is recommended instead of `@ts-ignore`. But even that should seldom be used.

Reply:
at this stage, I am trying to get it working first.